doc/dev/autograph.rst

```diff
@@ -852,3 +852,34 @@ rather than runtime.
 
 For more details, see the :ref:`compile-time vs. runtime <compile_time>`
 documentation.
+
+
+Adding modules for Autograph conversion
+---------------------------------------
+
+Due to the experimental state of Autograph support, the following modules 
+have been excluded from Autograph conversion: ``pennylane, catalyst, jax``. 
+However, it is possible to enable these or other previously excluded 
+modules via the ``autograph_include`` parameter, which provides a list 
```

Comment:
Continuing this line of reasoning, I would suggest phrasing this slightly differently, because the goal should really not be to override the disallow list in catalyst (dangerous), but to be able to provide exceptions for specific submodules for which conversion may be appropriate.

Reply:
👍